actors/market/src/deal.rs

```diff
@@ -133,7 +142,9 @@ impl Cbor for ClientDealProposal {}
 
 #[derive(Clone, Debug, PartialEq, Copy, Serialize_tuple, Deserialize_tuple)]
 pub struct DealState {
-    pub sector_start_epoch: ChainEpoch, // -1 if not yet included in proven sector
-    pub last_updated_epoch: ChainEpoch, // -1 if deal state never updated
-    pub slash_epoch: ChainEpoch,        // -1 if deal never slashed
+    pub sector_start_epoch: ChainEpoch,
```

Comment:
does rust really want these below the constant they refer to?  I think they belong above

Reply:
This is what rustfmt did to me, but you're right. Fixing.